back-end/dms/permissions.py

```diff
@@ -0,0 +1,10 @@
+from rest_framework import permissions
+
+
+class IsOwnerOrReadOnly(permissions.BasePermission):
```

Comment:
Permissions can be `OR`ed: `permission_classes = [IsOwner | ReadOnly]`
Bettter to split them.